cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -222,16 +220,15 @@ func runApply(flags *applyFlags) error {
 	}
 
 	fmt.Println("")
-	fmt.Printf("[upgrade/successful] SUCCESS! Your cluster was upgraded to %q. Enjoy!\n", flags.newK8sVersionStr)
+	fmt.Printf("[upgrade/successful] SUCCESS! Your cluster was upgraded to %q. Enjoy!\n", cfg.KubernetesVersion)
 	fmt.Println("")
 	fmt.Println("[upgrade/kubelet] Now that your control plane is upgraded, please proceed with upgrading your kubelets if you haven't already done so.")
 
 	return nil
 }
 
-// SetImplicitFlags handles dynamically defaulting flags based on each other's value
-func SetImplicitFlags(flags *applyFlags) error {
-	if len(flags.newK8sVersionStr) == 0 {
+func assertVersionStringIsEmpty(version string) error {
```

Comment:
1. This is missing a `Not` from its name (e. g. `assertVersionStringIsNotEmpty`)
2. I'd rather have the single if statement moved to the only place the func gets called. But that's only if you are not going to add more checks to it in a follow up PR(s).

WDYT?